src/main/java/me/modmuss50/dg/globe/VoidChunkGenerator.java

```diff
@@ -0,0 +1,64 @@
+package me.modmuss50.dg.globe;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.world.BlockView;
+import net.minecraft.world.ChunkRegion;
+import net.minecraft.world.Heightmap;
+import net.minecraft.world.WorldAccess;
+import net.minecraft.world.biome.source.BiomeSource;
+import net.minecraft.world.chunk.Chunk;
+import net.minecraft.world.gen.StructureAccessor;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.chunk.StructuresConfig;
+import net.minecraft.world.gen.chunk.VerticalBlockSample;
+
+public class VoidChunkGenerator extends ChunkGenerator {
+	// Just an example of adding a custom boolean
+	protected final boolean customBool;
```

Comment:
[NIT] Not needed here.

Reply:
Whoops, that happens when you follow the tutorial too close :smile:
Pushed a fix.